code/modules/mob/living/carbon/xenomorph/castes/hunter/abilities_hunter.dm

```diff
@@ -685,3 +762,148 @@
 	owner.playsound_local(owner, 'sound/effects/alien/new_larva.ogg', 25, 0, 1)
 	cooldown_duration = initial(cooldown_duration) //Reset the cooldown timer to its initial state in the event of a whiffed Silence.
 	return ..()
+
+//assassin things
+
+// ***************************************
+// *********** Lunge
+// ***************************************
+
+/datum/action/ability/activable/xeno/pounce/lunge
+	name = "Lunge"
+	action_icon = 'icons/Xeno/actions/hunter.dmi'
+	action_icon_state = "assassin_lunge"
+	desc = "Swiftly lunge at your destination, if on a target, attack and stun them briefly."
+	ability_cost = 10
+	cooldown_duration = 6 SECONDS
+	keybinding_signals = list(
+		KEYBINDING_NORMAL = COMSIG_XENOABILITY_HUNTER_LUNGE,
+	)
+	use_state_flags = ABILITY_USE_BUCKLED
+
+//Triggers the effect of a successful pounce on the target.
+/datum/action/ability/activable/xeno/pounce/lunge/trigger_pounce_effect(mob/living/living_target)
+	playsound(get_turf(living_target), 'sound/voice/alien/roar4.ogg', 25, TRUE)
+	var/mob/living/carbon/xenomorph/xeno_owner = owner
+	xeno_owner.UnarmedAttack(living_target)
+	step_away(living_target, xeno_owner, 1, 3)
+	xeno_owner.face_atom(living_target)
+	living_target.Stun(XENO_POUNCE_STUN_DURATION) // no disarm but standing stun to make it feel like you got bodyslammed by a xeno
```

Comment:
this whole ability is just normal pounce only significantly better. almost twice as spammable (you should be able to stun a target like 5-6 times in 10 seconds), does damage, and no knockdown means you can use it more safely since the victim will still block bullets.

Reply:
No knockdown also keeps their weapon in hand and all, the stun will be gone though, also victim is pushed away a tile and there happens to be a gap between hunter and target, that also would expose to bullets id say, regardless assassin is mega squishy anyway.

Reply:
now made it so there is no stun, does regular slash on bump, still dont know what you mean 5-6 times in 10 seconds, you can do this every 6 seconds though.

Reply:
You combo your stuns. Assassin doesn't break stealth on hit, so you can combo leap stuns with stealth attack stuns to just repeatedly stun people into oblivion